memote/suite/tests/test_annotation.py

```diff
@@ -70,6 +70,28 @@ def test_reaction_annotation_presence(read_only_model):
     assert len(ann["data"]) == 0, ann["message"]
 
 
+@annotate(title="Gene Products without Annotation", type="count")
```

Comment:
I'd like @Midnighter's opinion on this but I'd still call this 'genes' because that is the name of the object you look for and in theory, the 'gene product' is reflected by the gene-protein-reaction rules for each reaction.

Calling this gene product while the object itself is called gene may confuse users in my opinion.

Reply:
Changed to genes/gene-products

Reply:
I don't think it should be anything else but simply `gene` for the reasons stated above. Like I said, the 'gene-product' aspect is reflected only by the GPR.